### DISCUSSION TITLE
comma vs. pipe

### DISCUSSION THREAD

**Discussion by benfishbus** [meisnate12/Plex-Meta-Manager#578] [Q&A] [answered]
I noticed on the TMDb Builders wiki page:
<html><body>

without_keywords | Exclude items with certain keywords. You can comma and pipe separate these values to create an 'AND' or 'OR' logic. | String
-- | -- | --

</body>
</html>

Is this technique (comma=AND, pipe=OR) applicable anywhere multiple values are employed, or just here?

**Comment by meisnate12** [comment, marked as answer]
ill be honest with you idk because apparently the `|` works with `with_networks` and `,` does not even though tv discover says it uses `,` so idk and i dont have the time to test each individual attribute to see what works. so my best advice would be to try it and see if it works
